bin/generate-nav

```diff
@@ -1,19 +1,29 @@
 #!/usr/bin/env node
 
+const {docPagesDirectory, versions} = require('../constants');
 const fileProcessor = require('../src/lib/fileProcessor');
-const { readFileSync, writeFile } = require('fs');
+const { readFileSync, writeFile, readdirSync } = require('fs');
 const { resolve } = require('path');
 const jsyaml = require('js-yaml');
 
 const loadedOutline = jsyaml.safeLoad(readFileSync(`${__dirname}/../config/outline.yaml`, 'utf8'));
-const docs = { chapters: [] };
 
-loadedOutline.chapters.forEach((directory) => {
-    const items = directory.items.map(
-        file => fileProcessor.processFile(directory.path, file.id)[0]
-    );
+versions.map(version => {
+    let docs = { chapters: [] };
+    let versionDirectory = isNaN(version) ? version : `v${version}`;
 
-    docs.chapters.push({ ...directory, items });
-});
+    loadedOutline.chapters.forEach((directory) => {
+        let files = readdirSync(`${docPagesDirectory}${versionDirectory}/${directory.path}`).filter(file => file.endsWith('.md')).map(file => file.slice(0,-3));
+        directory.items = directory.items.filter(x => files.includes(x));
+        files = [...new Set(directory.items.concat(files))];
```

Comment:
Why do we need to do this? Does directory contain not all the files?

Reply:
see previous comment :)